Itera/HelloBrick
# Websocket interface

This intergrates the EV3 code with the websocket interface of the server application (https://github.com/havardh/kurs-the-flag-web).

To test:
 - Connect the computer and robot to the same wifi (usb and bluetooth will also work)
 - Set the HOST variable in RobotState to the IP of computer
 - Run the server app and the this branch on the EV3
 - Open the simulation page on the web app (localhost:3000/simulate)
 - enter the IP of the EV3 (printed by the server app when the EV3 has connected)
 - Clicking on the color buttons should send a Status message to the EV3.
